SNT/TypeGenerator.cs

```diff
@@ -0,0 +1,800 @@
+
+
+
+
+
+using System;
+using System.Linq;
+using System.Numerics.Tensors;
+using TorchSharp;
+
+namespace Torch.SNT 
+{
+
+    public sealed class ByteTorchTensor : DenseTensor<byte>
```

Comment:
Please add xml docs to all `public` types, methods and fields.

Reply:
Fixed.